paper-input-container.html

```diff
@@ -453,20 +458,21 @@
 
     _onBlur: function() {
       this._setFocused(false);
+      this._handleValue(this._inputElement, true);
     },
 
     _onInput: function(event) {
-      this._handleValue(event.target);
+      this._handleValue(event.target, true);
     },
 
     _onValueChanged: function(event) {
-      this._handleValue(event.target);
+      this._handleValue(event.target, true);
     },
 
-    _handleValue: function(inputElement) {
-      var value = inputElement[this._propertyForValue] || inputElement.value;
+    _handleValue: function(inputElement, validateNow) {
```

Comment:
Once again, I'm not super excited about the boolean argument. It's kind of weird to read on lines 461, 465, and 469 for example. I'm not sure if there is a better way, though.